api-2.2/pom.xml

```diff
@@ -79,6 +79,16 @@
                 <filtering>true</filtering>
             </testResource>
         </testResources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Do we need this?

Reply:
Yes, support for java 8 lambda 
example used this:
 `patientList.stream().forEach(patient -> {
				conditions.addAll(conditionService.getActiveConditions(patient)
						.stream()
						.map(this::generateFHIRCondition)
						.collect(Collectors.toList()));
			});`